developer/introduction.rst

```diff
@@ -0,0 +1,258 @@
+Introduction to Python and to the Django world
+==============================================
+
+This page will introduce you to python, to some of python's modules such
+as ipython and to the django and django-rest frameworks.
```

Comment:
we should make the goal of this page clearer, we aim at helping technical users and contributors to work on OpenWISP and we introduce them to some python tools they can find very useful during development and debugging